manifests/course/puppetize.pp

```diff
@@ -17,8 +17,16 @@
     include classroom::master::dependencies::dashboard
     include classroom::master::hiera
 
+    $base_plugin_list = [ "Certificates", "Classification", "ConsoleUser", "Docker", "Logs", "Dashboard", "CodeManager", "ShellUser", "Gitviz" ]
```

Comment:
Shouldn't be difficult to refactor the base list into a containing class, I guess. PI only has the extra Gitviz plugin and the other two are the same.